sbin/submitty_autograding_worker.py

```diff
@@ -190,6 +190,8 @@ def read_autograding_worker_json():
             #grab the key and the value. NOTE: For now there should only ever be one pair.
             name = list(name_and_stats.keys())[0]
             stats = name_and_stats[name]
+    except FileNotFoundError as e:
+        raise SystemExit("autograding_worker.json not found. Have you registered this worker with a Submitty host yet?") from e
```

Comment:
Here's a sample of how this looks behind `journalctl -u submitty_autograding_worker`:

```
Jul 02 05:30:51 submitty-worker-bionic-vm systemd[1]: submitty_autograding_worker.service: Main process exited, code=exited, status=1/FAILURE
Jul 02 05:30:51 submitty-worker-bionic-vm systemd[1]: submitty_autograding_worker.service: Failed with result 'exit-code'.
Jul 02 05:30:51 submitty-worker-bionic-vm systemd[1]: submitty_autograding_worker.service: Service hold-off time over, scheduling restart.
Jul 02 05:30:51 submitty-worker-bionic-vm systemd[1]: submitty_autograding_worker.service: Scheduled restart job, restart counter is at 5.
Jul 02 05:30:51 submitty-worker-bionic-vm systemd[1]: Stopped Submitty Autograding Worker.
Jul 02 05:30:51 submitty-worker-bionic-vm systemd[1]: Started Submitty Autograding Worker.
Jul 02 05:30:52 submitty-worker-bionic-vm submitty_autograding_worker.py[29144]: cleaned up old jobs
Jul 02 05:30:52 submitty-worker-bionic-vm submitty_autograding_worker.py[29144]: autograding_worker.json not found. Have you registered this worker with a Submitty host yet?
```